app/move/steps/create.js

```diff
@@ -55,7 +55,14 @@ const riskStep = {
     {
       field: 'from_location_type',
       value: 'prison',
-      next: 'special-vehicle',
+      next: [
+        {
+          field: 'to_location_type',
+          value: 'prison',
+          next: 'save',
```

Comment:
Does the prison to prison journey not collect any information about health affecting transport then?

Reply:
Ah right. Does the risk step come after the health step in this journey? Is there a specific reason if so? 

If not, I wonder if we keep the ordering of court -> risk (/release status) -> health (/special vehicle) consistent to simplify some of this?

Reply:
The sequence is:
- who
- where
- when
- move details (includes the reason for the move, but not the risk information, which is not captured)
- move agreed, and by who
- special vehicle
- save

Reply:
It's not actually that much of a complication on the existing journeys. The mistaken change on the `riskStep` might have given the impression that there's more variation than there is.

Reply:
I see, yeah I thought the risk step was still part of the journey.